### PR TITLE
Adding Space IoT article

### PR DESCRIPTION
**Pull request by fshmcallister**
Should all be complete, builds on my own machine happily.
Have discussed with Al and been given the green light.
Typos on my end have been removed. Let me know if there's others that you have spotted.

### CODE REVIEW

File: _posts/2019-11-15-space-iot.md [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+category: [news,thoughts]
+author: Fiona McAllister
+tags: [Open Source]
+class: post-template
+---
+
+## Introduction
+
+In a world full of buzzwords, you’ve probably already come across the term ‘New Space.’ But, what exactly is it and why should you as developers care about it? WoTT is a security company aimed at Linux distros to secure server to receiver connections in an easy, flexible way. So where do we fit?
+In simple terms, because developers are needed in all aspects of commerce- but more so, the opportunity to capitalise early on a sector that is estimated to be worth [1.1 trillion dollars](https://www.nato-pa.int/download-file?filename=sites/default/files/2018-12/2018%20-%20THE%20FUTURE%20OF%20SPACE%20INDUSTRY%20-%20BOCKEL%20REPORT%20-%20173%20ESC%2018%20E%20fin.pdf) by 2040 (and sitting on a comfortable 360 billion now). So much of the focus in the space sector up til now has been setting up infrastructure.
```
Comment (vpetersson) [inline _posts/2019-11-15-space-iot.md]: Formatting.

File: _posts/2019-11-15-space-iot.md [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+And it’s not as if the space sector is an ‘alien’ endeavour for developers - in fact in many ways space and IoT have always gone hand in hand.
+
+## The Past
+
+Manned spaceflight accounts for a very small portion of the space sector as a whole and has typically been motivated by military pursuit. The most 'popular' space endeavour, the ISS, has only been manned by crew since 2000 (assembly beginning in 1998).
+
+{% asset blog/space-economy.png srcset:width="1300 2x" srcset:width="650 1x" alt="Global Space Sector" %}
+*in this image, you can consider manned spaceflight as a part of 'launch', graph found [here](https://www.sia.org/wp-content/uploads/2019/05/2019-SSIR-2-Page-20190507.pdf)*
+
+Generally, most experiments related to space are done remotely. Many of the experiments present on the ISS contain data that needs to be broadcasted to Earth. In short, many of these experiments- which also need to be relatively low power- can be considered analagous to IoT problems. The parallels have always been there. As the use of internet as a medium for transferring data in the space domain grows, so too will the need for security. Companies like WoTT can help broach that gap.
```
Comment (vpetersson) [inline _posts/2019-11-15-space-iot.md]: Again - formatting with the dashes.

File: _posts/2019-11-15-space-iot.md [hunk 3]
```diff
@@ -0,0 +1,52 @@
...
+
+## The Now
+
+We've since moved on from the days of the Space Race and the military era to where we are currently- a more exciting commerical branch of the sector: New Space. It's here where prospective developers can really shine with many commercial start ups looking to outsource several components (such as security).
```
Comment (vpetersson) [inline _posts/2019-11-15-space-iot.md]: Dash again.

File: _posts/2019-11-15-space-iot.md [hunk 4]
```diff
@@ -0,0 +1,52 @@
...
+
+We now have the opportunity to interface with projects in space- and not all of them are scientific. There are currently [3 projects](https://www.esa.int/Science_Exploration/Human_and_Robotic_Exploration/Research/ICE_Cubes_space_research_service_open_for_business) onboard the ISS done in collaboration with ESA and the International Space University (ISU); one of which is an art project (a potential area of commerical growth). Users on Earth can interface and interact with this in near real-time. These projects are set up by the onboard astronauts and essentially left to their own devices. Their data gets broadcast to Earth via the internet - much like an IoT device. It's natural to assume more projects like the ICE cubes will become more prominent in the future. As this sector increases, so does its vulnerability; and as we know IoT on Earth is already a poorly secured domain - remember the [Samsung fridges](https://www.pentestpartners.com/security-blog/hacking-ruining-the-samsung-smart-fridge/)?
```
Comment (vpetersson) [inline _posts/2019-11-15-space-iot.md]: And here.

File: _posts/2019-11-15-space-iot.md [hunk 5]
```diff
@@ -0,0 +1,52 @@
...
+
+We often take cybersecurity for granted in the developer world, but it's often not even considered by the everyday person; even in scientfic roles. How many of these experiments for example contain [common vulnerabilities]({{site.url}}blog/tutorials/2019/11/04/cve) (which we here at WoTT can help you detect) in their hardware? The demand for workers with a technical background in the space sector is increasing along with these risks. 
+
+Additionally, IoT itself on Earth is expanding at an exponential rate and we simply don't have the infrastructure to support it. There are growing talks in implementing LEO (Low Earth Orbit) satellite constellations as a means of supplying low-latency internet everywhere- this is particularly relevant with the increase of prioritisation of smart cities globally. Satellites can reach corners of the globe that conventional internet supplying methods just can't. [IoTEE](https://cordis.europa.eu/project/rcn/207924/brief/en) is an EU funded project concerned with providing such services to IoT devices specifically, and it is going to be tested on the ISS.
```
Comment (vpetersson) [inline _posts/2019-11-15-space-iot.md]: Again ;)

**Comment by vpetersson** [comment]
@fshmcallister LGTM. Just fix the dashes and update the date.

**Comment by fshmcallister** [comment]
Where on earth are these dashes from?!

**Pull request closed without merging**